rules/impl.bzl

```diff
@@ -63,7 +64,7 @@ def _run_android_lint(
         android_lint_skip_bytecode_verifier: Disables bytecode verification
     """
     inputs = []
-    outputs = [output]
+    outputs = [output, html_output]
```

Comment:
It would be great to support customizing the output format so that the user can pick which formats they want. Having to write them all out all the time also seems inefficient.

Reply:
Should we rename --output to --xml-output then to be explicit? 

I'm thinking that we should flag it in the entrypoint .bzl: 

html_output = True
xml_output = True